pkg/build/api/v1/types.go

```diff
@@ -622,6 +622,19 @@ type BuildOutput struct {
 	// up the authentication for executing the Docker push to authentication
 	// enabled Docker Registry (or Docker Hub).
 	PushSecret *kapi.LocalObjectReference `json:"pushSecret,omitempty" protobuf:"bytes,2,opt,name=pushSecret"`
+
+	// labels define a list of labels that are applied to the resulting image. If there are
+	// multiple labels with the same name then the last one in the list is used.
+	Labels []ImageLabel `json:"labels,omitempty" protobuf:"bytes,3,rep,name=labels"`
```

Comment:
it's confusing that these will result in docker labels, not API labels on the object that the `to` reference points to